app/cdap/components/Replicator/Create/Content/SelectColumns/index.tsx

```diff
@@ -384,6 +384,7 @@ class SelectColumnsView extends React.PureComponent<ISelectColumnsProps, ISelect
                       <Checkbox
                         color="primary"
                         className={classes.radio}
+                        disabled={this.state.primaryKeys.indexOf(row.name) !== -1}
```

Comment:
Can you check if you click on the select all at the table header, will it deselect the table? I think it will.